### DISCUSSION TITLE
Running multiple tests within one test

### DISCUSSION THREAD

**Discussion by ThatTonybo** [avajs/ava#2857] [Q&A]
I have a test as follows:

```js
test('Check that an item exists using has() method', async t => { ... });
```

The `has()` method I'm testing accepts three distinct params, which I'd like to test. Two will return the same result, and one will return a different result. 

Am I able to do multiple `t.deepEqual` or `t.is` in a row before the test will pass, or will the test pass on the first and not continue?

**Comment by novemberborn** [comment]
AVA runs all assertions in your test. If there are multiple failures you'll only see the first.

Whether you write three different test for the parameters, or one with multiple assertions, is up to you.
